mirdata/cante100.py

```diff
@@ -0,0 +1,491 @@
+        if track_id not in DATA.index:
+            raise ValueError('{} is not a valid track ID in Example'.format(track_id))
+
+        self.track_id = track_id
+
+        if data_home is None:
+            data_home = utils.get_default_dataset_path(DATASET_DIR)
+
+        self._data_home = data_home
+
+        self._track_paths = DATA.index[track_id]
+        self.audio_path = os.path.join(
+            self._data_home, self._track_paths['audio'][0]
+        )
+        self.spectrum_path = os.path.join(
+            self._data_home, self._track_paths['spectrum'][0]
+        )
+        self.f0_path = os.path.join(self._data_home, self._track_paths['f0'][0])
+        self.notes_path = os.path.join(self._data_home, self._track_paths['notes'][0])
+
+        metadata = DATA.metadata(data_home=data_home)
+        if metadata is not None and track_id in metadata:
+            self._track_metadata = metadata[track_id]
+        else:
+            self._track_metadata = {
+                'musicBrainzID': None,
+                'artist': None,
+                'title': None,
+                'release': None,
+                'duration': None,
+            }
+
+        self.identifier = self._track_metadata['musicBrainzID']
+        self.artist = self._track_metadata['artist']
+        self.title = self._track_metadata['title']
+        self.release = self._track_metadata['release']
+        self.duration = self._track_metadata['duration']
+
+    @property
+    def audio(self):
+        """(np.ndarray, float): audio signal, sample rate"""
+        return load_audio(self.audio_path)
+
+    @property
+    def spectrum(self):
+        """(np.ndarray, float): spectrum"""
+        return load_spectrum(self.spectrum_path)
+
+    @utils.cached_property
+    def melody(self):
+        """F0Data: audio signal, sample rate"""
+        return load_melody(self.f0_path)
+
+    @utils.cached_property
+    def notes(self):
+        """NoteData: audio signal, sample rate"""
+        return load_notes(self.notes_path)
+
+    def to_jams(self):
+        """Jams: the track's data in jams format"""
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path,
+            spectrum_cante100_path=self.spectrum_path,
+            f0_data=[(self.melody, 'pitch_contour')],
+            note_data=[(self.notes, 'note_hz')],
+            metadata=self._track_metadata,
+        )
+
+
+def load_spectrum(spectrum_path):
+    """Load a cante100 dataset spectrum file.
+
+    Args:
+        spectrum_path (str): path to audio file
+
+    Returns:
+        np.array: spectrum
+
+    """
+    if not os.path.exists(spectrum_path):
+        raise IOError("audio_path {} does not exist".format(spectrum_path))
+
+    with open(spectrum_path, 'r') as csvfile:
+        reader = csv.reader(csvfile, delimiter=' ', quotechar='\n')
+
+        spectrum_ = []
+        total_rows = 0
+        for row in reader:
+            spectrum_.append(row[:514])
```

Comment:
I don't understand why 514 is hard coded. Is there the case where there are more lines? 
anyway, if you do the genfromtxt then this line will dissapear